src/components/MyProjects/index.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react'
```

Comment:
__[eslint]__
* Error - Unable to resolve path to module 'react'. (import/no-unresolved)

<!-- Comment by [SideCI](https://sideci.com) -->